Drakhtar/GameObjects/RecruitmentStat.cpp

```diff
@@ -50,9 +49,15 @@ std::string RecruitmentStat::fillText() {
   text += "Speed-> " + std::to_string(stats.speed) + "\n";
   text += "Attack range-> " + std::to_string(stats.attackRange) + "\n";
   text += "Move range-> " + std::to_string(stats.moveRange) + "\n";
+  text += "Prize-> " + std::to_string(stats.prize * 2) + "\n";
   return text;
 }
 
+std::string RecruitmentStat::infoFillText() {
```

Comment:
```suggestion
std::string RecruitmentStat::infoFillText() const {
```

Reply:
It was supposed to be removed, alas I forgot